python/llm/example/transformers/transformers_int4/chatglm/README.md

```diff
@@ -2,13 +2,17 @@
 
 In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on ChatGLM models. For illustration purposes, we utilize the [THUDM/chatglm-6b](https://huggingface.co/THUDM/chatglm-6b) as a reference ChatGLM model.
 
-## 0. Requirements
+> **Note**: If you want to download the model, please refer to [here](https://huggingface.co/docs/hub/models-downloading#using-git). And you don't need to convert the original [THUDM/chatglm-6b](https://huggingface.co/THUDM/chatglm-6b) model.
```

Comment:
Change `And you don't need to convert ...` to something like we will do the conversion and quantization at runtime?